packages/components/organisms/f-checkout/test/component/f-checkout-guest.component.desktop.spec.js

```diff
@@ -29,17 +29,20 @@ describe('f-checkout "guest" component tests - @browserstack', () => {
         expect(pathname).toEqual(loginPath);
     });
 
-    it('should prevent a user from writing a note of over 200 characters', () => {
+    forEach([
+        ['firstName', 100],
+        ['lastName', 100],
+        ['userNote', 200]
+    ])
+    .it('should prevent a user from entering more characters than allowed in a field', (field, maxlength) => {
```

Comment:
Could set the description as:
`should prevent a user from entering a "%s" with more than "%s" characters`